patching_todo.txt

```diff
@@ -112,16 +112,9 @@
 //}
 	
 	//Lighting:
-		//Re-add TileLoader.NearbyEffects calls (perhaps multiple) to somewhere in SceneMetrics
-		
 		//Re-add this MusicBox logic, to somewhere in SceneMetrics:
 			if (TileLoader.IsModMusicBox(tile) && tile.frameX >= 36)
 				Main.musicBox = SoundLoader.tileToMusic[tile.type][tile.frameY / 36 * 36];
-		
-		//Call this at the end of SceneMetrics.ScanAndExportToMain:
-			WorldHooks.TileCountsAvailable(_tileCounts);
-		
-		//Resize the arrays that ScreenMetrics.UpdateOreFinderData uses.
```

Comment:
If anyone is curious why I deleted this line even though I didn't patch it, it was already done way back on 1.3, `UpdateOreFinderData` just uses the same arrays as before.